builder/tencentcloud/cvm/access_config.go

```diff
@@ -64,6 +64,12 @@ type TencentCloudAccessConfig struct {
 	Zone string `mapstructure:"zone" required:"true"`
 	// Do not check region and zone when validate.
 	SkipValidation bool `mapstructure:"skip_region_validation" required:"false"`
+	// The endpoint you want to reach the cloud endpoint,
+	// if tce cloud you should set a tce cvm endpoint.
+	CvmEndpoint string `mapstructure:"cvm_endpoint" required:"false"`
+	// The endpoint you want to reach the cloud endpoint,
+	// if tce cloud you should set a tce vpc endpoint.
+	VpcEndpoint string `mapstructure:"vpc_endpoint" required:"false"`
```

Comment:
According to the comment, it looks to me that if one is specified, the other one should be as well?
If that's the case, we can probably add this check to the `Prepare` function so we can return a clear error message about the missing endpoint attribute in the config.

Otherwise, if I misunderstood the intent, we should probably amend the comments to make it clear what is required and what is not for enabling TCE.